### PR TITLE
Schema support mixed Chinese and English

### LINKED ISSUE #3426: schema supports mixed Chinese and English
The schema supports Chinese characters, but cannot be mixed with English characters, digits, and underscores.
It is recommended to support mixing them, such as `CREATE SPACE `测试空间1` (vid_type=FIXED_STRING(30));`.

image

### PR DESCRIPTION
**Pull request by heroicNeZha**
#### What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### What does this PR do?
Schema support mixed Chinese and English
```
# Directly support utf-8 / English / Number
CREATE TAG  队伍(名s字ss1 string);

# Using backquote support UTF-8 / English / Number / Full-width Charactor / Blank
CREATE TAG `队   伍；`(`名s字ss1` string);
```

#### Which issue(s)/PR(s) this PR relates to?
close #3426 
  
#### Checklist：
- [x] Documentation affected （Please add the label if documentation needs to be modified.)
- [ ] Incompatible （If it is incompatible, please describe it and add corresponding label.）
- [ ] Need to cherry-pick （If need to cherry-pick to some branches, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes：
Please confirm whether to reflect in release notes and how to describe:
>                                                                 `

### CODE REVIEW

File: src/parser/parser.yy [hunk 1]
```diff
@@ -406,7 +406,7 @@ static constexpr size_t kCommentLengthLimit = 256;
 
 name_label
     : LABEL { $$ = $1; }
-    | CHINESE_LABEL { $$ = $1; }
+    | UTF8_LABEL { $$ = $1; }
```
[the remark this answers is not in the document]
Comment (heroicNeZha) [reply src/parser/parser.yy]: done

**Review by CPWstatic — DISMISSED** [review]

File: src/parser/scanner.lex [hunk 2]
```diff
@@ -471,17 +473,6 @@ CHINESE_LABEL               ({U2}{U}|{U3}{U}{U}|{U4}{U}{U}{U})+
                                 // Must match /* */
                                 throw GraphParser::syntax_error(*yylloc, "unterminated comment");
                             }
-\`{CHINESE_LABEL}\`         {
```
Comment (yixinglu) [inline src/parser/scanner.lex:474]: Where is the guarantee that Chinese characters must be surrounded by backquote?

Comment (heroicNeZha) [reply src/parser/scanner.lex:474]: Both neo4j and mysql can use Chinese characters directly, do we have to use backquote?

Comment (CPWstatic) [reply src/parser/scanner.lex:474]: Why Chinese characters must be surrounded by backquote?

Comment (yixinglu) [reply src/parser/scanner.lex:474]: OK. no problem. I checked the two methods are also supported in mysql.

**Review by CPWstatic — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**